wrappers/services_gen_test.go

```diff
@@ -375,21 +374,48 @@ WantedBy=multi-user.target
 					Snap:   &snap.Info{SuggestedName: "snap"},
 					Daemon: "forking",
 				},
+				"baz": {
+					Name:   "baz",
+					Snap:   &snap.Info{SuggestedName: "snap"},
+					Daemon: "forking",
+				},
 			},
 		},
 		Name:        "app",
 		Command:     "bin/foo start",
 		Daemon:      "simple",
-		Before:      []string{"foo"},
-		After:       []string{"bar", "zed"},
 		StopTimeout: timeout.DefaultTimeout,
 	}
 
-	generatedWrapper, err := wrappers.GenerateSnapServiceFile(service)
-	c.Assert(err, IsNil)
+	for _, tc := range []struct {
+		after           []string
+		before          []string
+		generatedAfter  string
+		generatedBefore string
+	}{{
+		after:           []string{"bar", "zed"},
+		generatedAfter:  "snap.snap.bar.service snap.snap.zed.service",
+		before:          []string{"foo", "baz"},
+		generatedBefore: "snap.snap.foo.service snap.snap.baz.service",
+	}, {
+		after:           []string{"bar"},
+		generatedAfter:  "snap.snap.bar.service",
+		before:          []string{"foo"},
+		generatedBefore: "snap.snap.foo.service",
+	},
+	} {
+		c.Logf("tc: %v", tc)
+		service.After = tc.after
+		service.Before = tc.before
+		generatedWrapper, err := wrappers.GenerateSnapServiceFile(service)
+		c.Assert(err, IsNil)
 
-	c.Logf("service: \n%v\n", string(generatedWrapper))
-	c.Assert(string(generatedWrapper), Equals, expectedService)
+		expectedService := fmt.Sprintf(expectedServiceFmt, mountUnitPrefix, mountUnitPrefix,
+			tc.generatedAfter, tc.generatedBefore, "on-failure", "simple")
+
+		c.Logf("service: \n%v\n", string(generatedWrapper))
```

Comment:
I usually add it when there's more Assert()/Check() statements so that you don't have to add Commentf() everywhere. Though, I see now it's just one Assert() here. I'll force push a quick fix

Reply:
Heh, I clearly need more coffee. I'll drop the line.